packit_service/worker/helpers/build/babysit.py

```diff
@@ -222,6 +225,19 @@ def update_copr_builds(build_id: int, builds: Iterable["CoprBuildTargetModel"])
 
     logger.info(f"The status of {build_id} is {build_copr.state!r}.")
 
+    if (
+        srpm_build := SRPMBuildModel.get_by_copr_build_id(build_id)
+    ) and srpm_build.status == BuildStatus.pending:
+        build_copr_srpm = copr_client.build_proxy.get_source_chroot(build_id)
```

Comment:
shouldn't this be wrapped in the try-except as well?

Reply:
It probably should, but in that case I should do it also here, right?
https://github.com/packit/packit-service/blob/710e6942e811711b081915684a828fd95f98fcf4/packit_service/worker/helpers/build/babysit.py#L242

Reply:
I would say so

Reply:
Does it make sense like this?